docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -311,13 +311,54 @@ Returns language translation keys as JSON string.
 
 ### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)

```

Comment:
@aglensmith, linking these headers does not give context into what kind of information the link is leading to. The links should be separated out similar to the Cornerstone example links.

Reply:
I agree with @uwikaiddi. I think we need to change the format of this article and separate the names of the helpers and the associated links. 
```suggestion
### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)

```

Reply:
@uwikaiddi that's not related to this specific fix that nate requested, so it needs to go into a separate ticket.